### DISCUSSION TITLE
How to farm many machines with different keys in one main machine?

### DISCUSSION THREAD

**Discussion by acewinner1999** [Chia-Network/chia-blockchain#5167] [Support]
I have 5 machine with 5 different keys.
I want to harvest on my main machine.
When I inport my main machine ca and set the farm ip to other machine then run chia start harvester the main machine log show the connection peer bug on the Plots Passed Filter Tab it shows 0/0.
May I ask how to farm mutli machine with different keys to one main machine full_node?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
